sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -495,13 +495,15 @@ case class TruncateTableCommand(
 }
 
 /**
- * Command that looks like
+ * Commands can of the following forms.
  * {{{
  *   DESCRIBE [EXTENDED|FORMATTED] table_name partitionSpec?;
+ *   DESCRIBE [QUERY] <query>
```

Comment:
nit: `DESCRIBE [QUERY] query`? We need `<>`?

Reply:
If its under the bracket , it means optional isn't it ? Like in the previous example `extended ` and `formatted` are optional ?

Reply:
Yea, `[QUERY]` means optional. You mean `<query>` means optional, too?

Reply:
@maropu Sorry .. i misinterpreted. Here , i am just trying to say that they have to specify a query. Its not optional. How should i say it ?
```
DESCRIBE [QUERY] query_statement
```

Reply:
yea, yes.